### PR TITLE
EEPROM options for BTT SKR 1.4

### PR DESCRIPTION
**Pull request by radek8**
Addition of parameters for SKR V1.4 to enable EEPROM simulation on SD card, or enable additional module EEPROM I2C - 256k

### CODE REVIEW

File: Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_4.h [hunk 1]
```diff
@@ -31,6 +31,22 @@
...
+
+//
+//EEPROM I2C 
+//
+// uncomment to enable the EEPROM I2C module
+//#define I2C_EEPROM
+//#ifdef E2END
+// #undef E2END
+//#endif
+//#define E2END 0x7FFF // koncová adresa EEPROM 24C256 (32 kB = 8 kB)
```
Comment (thisiskeithb) [inline Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_4.h]: You should use `MARLIN_EEPROM_SIZE` instead.

**Comment by thisiskeithb** [comment]
Would this be more suited for the [`*_common`](https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.0.x/Marlin/src/pins/lpc1768/pins_BTT_SKR_common.h) file instead?

**Comment by radek8** [comment]
> Would this be more suited for the [`*_common`](https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.0.x/Marlin/src/pins/lpc1768/pins_BTT_SKR_common.h) file instead?

It would be possible but unnecessary.
pin_BTT_SKR _common.h is compatible for SKR V1.1 V1.3 V1.4 and V1.4Turbo.
SKR V1.1 and V1.3 do not have an I2C connector.
pins_BTT_SKR_V1_4.h is also compatible for Turbo, both have an I2C connector.

File: Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_4.h (hunk 1 above)
Comment (radek8) [reply Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_4.h]: Yes, tested.
#define MARLIN_EEPROM_SIZE 0x8000 // 32Kb (24C256)
both variants are functional.
So you recommend to convert to MARLIN_EEPROM_SIZE?

Comment (thisiskeithb) [reply Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_4.h]: > So you recommend to convert to MARLIN_EEPROM_SIZE?

Yes

**Comment by radek8** [comment]
Tested and works well. Thanks for editing

**Pull request merged**